File: backend-repo/FaniverseBE/src/main/java/fantastic/faniverse/product/dto/ProductDto.java
package fantastic.faniverse.product.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class ProductDto {
    private Long id;
    private String title;
    private String content;
    private String category;
    private String imageUrl;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;

    // Constructors
    public ProductDto(Long id, String title, String content, String category, String imageUrl, LocalDateTime createdAt, LocalDateTime updatedAt) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.category = category;
        this.imageUrl = imageUrl;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
    }
}

File: frontend-repo/FaniverseFE2/app/src/main/java/com/example/login/model/ProductDto.kt
package com.example.login.model

import java.time.LocalDateTime

data class ProductDto(
    val id: Long,
    val title: String,
    val content: String,
    val category: String,
    val imageUrl: String,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime
)
